platform/javascript/js/engine/preloader.js

```diff
@@ -32,7 +32,8 @@ const Preloader = /** @constructor */ function () { // eslint-disable-line no-un
 			loaded: 0,
 			done: false,
 		};
-		return fetch(file).then(function (response) {
+		var addCacheBuster=typeof CACHE_BUSTER=="string"?CACHE_BUSTER:"";
```

Comment:
Use `const` for a variable that's only assigned once, and use `===` instead of `==`:

```suggestion
		const addCacheBuster = typeof CACHE_BUSTER === "string" ? CACHE_BUSTER : "";
```